### ISSUE TITLE
ENH: Random eye movement when idle

### ISSUE THREAD

**Issue by jschrempp** [BobGlicksman/Animatronics#46]
To add some interest, we should have the eyes open and look around randomly when the TOF sensor detects nothing. This behavior would help pull people in to see the puppet.

**Comment by BobGlicksman** [comment]
Yes, and perhaps send a new event to the mouth when this happens.  So the mouth could play a clip, e.g. snore.
